parlai/tasks/visdial/build.py

```diff
@@ -41,19 +43,45 @@ def build(opt):
 
     if not build_data.built(dpath):
         print('[building data: ' + dpath + ']')
+
         build_data.remove_dir(dpath)
         build_data.make_dir(dpath)
 
         # Download the data.
         fname1 = 'visdial_0.9_train.zip'
         fname2 = 'visdial_0.9_val.zip'
-
+       
         url = 'https://computing.ece.vt.edu/~abhshkdz/data/visdial/'
         build_data.download(url + fname1, dpath, fname1)
         build_data.download(url + fname2, dpath, fname2)
 
         build_data.untar(dpath, fname1)
         build_data.untar(dpath, fname2)
+        
+        print('processing unpacked files')
+        # Use 1000 examples from training set as validation.
+        json1 = os.path.join(dpath, fname1.rsplit('.', 1)[0] + '.json')
+        with open(json1) as t_json:
+            train_data = json.load(t_json)
+
+        random.seed()
```

Comment:
actually can you fix this so it's the same valid set every time? e.g. `random.seed(42)`